food_segmenter.ipynb

```diff
@@ -0,0 +1,193 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "provenance": [],
+      "include_colab_link": true
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/lukeschaller/Take_a_Byte/blob/main/food_segmenter.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "zqowkrSNDHa_"
+      },
+      "outputs": [],
+      "source": [
+        "import tensorflow.compat.v2 as tf\n",
+        "import tensorflow_hub as hub\n",
+        "import numpy as np\n",
+        "from PIL import Image"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "categories = [\"background\", \"vegetables | leafy_greens\", \"vegetables | stem_vegetables\", \"vegetables | non-starchy_roots\", \"vegetables | other\", \"fruits\", \"protein | meat\", \"protein | poultry\", \"protein | seafood\", \"protein | eggs\", \"protein | beans/nuts\", \"starches/grains | baked_goods\", \"starches/grains | rice/grains/cereals\", \"starches/grains | noodles/pasta\", \"starches/grains | starchy_vegetables\", \"starches/grains | other\", \"soups/stews\", \"herbs/spices\", \"dariy\", \"snacks\", \"sweets/desserts\", \"beverages\", \"fats/oils/sauces\", \"food_containers\", \"dining_tools\", \"other_food\"]"
+      ],
+      "metadata": {
+        "id": "AuP7Pn4fgjHr"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "model_path = \"/content/Model1\"\n",
+        "model = tf.saved_model.load(model_path)"
```

Comment:
OSError: SavedModel file does not exist at: [/content/Model1/](https://vscode-remote+codespaces-002bobscure-002dtribble-002dppw5wjrp6gvh9645.vscode-resource.vscode-cdn.net/content/Model1/){saved_model.pbtxt|saved_model.pb}

Is there any way this can be fixed in this same commit? I think having the model in this commit might solve this